app/controllers/v0/messages_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+module V0
+  class MessagesController < HealthcareMessagingController
+    def index
+      folder_id = params[:folder_id].try(:to_i)
+      response = client.get_folder_messages(folder_id, 1, 100)
```

Comment:
Was thinking ahead to when we have to implement search and filtering. It will be necessary to have the entire result set. But yes, these page and page_size should be parameters.